app/javascript/stimulus/controllers/article_controller.js

```diff
@@ -0,0 +1,32 @@
+import { Controller } from 'stimulus';
```

Comment:
I didn't notice before but why stimulus in the path name? Logically these should just be in `app/javascript/internal` ? The fact that we're using stimulus shouldn't matter in the path

Reply:
I found it useful as a flag because the vast majority of our application is using Preact. I'm alright with changing this though if others don't find it useful.

Reply:
I found it weird to have code grouped by framework, not by functionality, that's why!